derp/derphttp/derphttp_client.go

```diff
@@ -30,6 +30,9 @@ import (
 	"tailscale.com/types/logger"
 )
 
+// DefaultTLSConfig is the base *tls.Config used by a DERP Client.
+var DefaultTLSConfig = &tls.Config{}
```

Comment:
I'd prefer if this were moved to an optional exported field on Client, just named TLSConfig. If nil, automatic.